src/contribute/contribution-guide.md

```diff
@@ -122,7 +122,10 @@ To configure annotation processing in IntelliJ:
 ###### Checkstyle
 IntelliJ supports checkstyle within the IDE using the Checkstyle-IDEA plugin.
 
-1. Install the "Checkstyle-IDEA" plugin from the IntelliJ plugin repository.
+1. Install the "Checkstyle-IDEA" plugin
+    1. download version 5.3.1 from [Checkstyle-IDEA versions](https://plugins.jetbrains.com/plugin/1065-checkstyle-idea)
```

Comment:
First letter in sentence should be capitalized. (Same for next couple of sentences.